pkg/scheduler/framework/interface.go

```diff
@@ -284,7 +284,7 @@ func (s *Status) Equal(x *Status) bool {
 	if !cmp.Equal(s.reasons, x.reasons) {
 		return false
 	}
-	return cmp.Equal(s.plugin, x.plugin)
+	return cmp.Equal(s.plugins, x.plugins)
```

Comment:
I think we should use [cmpopts.SortSlices](https://godoc.org/github.com/google/go-cmp/cmp/cmpopts#SortSlices) for both plugins and reasons.
For plugins, we don't care their orders.
For reasons, currently we append them from a map which will also make it unordered.
However, we seems only to use `Equal` in tests, I wonder if some user is using it and got affected.